packages/fluentui/docs/src/global.d.ts

```diff
@@ -25,3 +25,9 @@ declare interface Window {
   resetExternalLayout?: () => void;
   switchTheme?: (themeName: string) => void;
 }
+
+declare module 'csstype' {
+  interface Properties {
+    msGridRow?: any;
```

Comment:
Please could we be a bit more specific?
Would this
```suggestion
    msGridRow?: number | string;
```
cover all the cases?